packages/metal-incremental-dom/src/incremental-dom.js

```diff
@@ -20,8 +18,10 @@ import { globals } from 'metal';
  */
 
 (function (global, factory) {
-  (factory((global.IncrementalDOM = global.IncrementalDOM || {})));
-}(globals.window, function (exports) { 'use strict';
+  typeof exports === 'object' && typeof module !== 'undefined' ? factory(exports) :
```

Comment:
These lines will probably cause some problems with some build types we support (I believe that for gulp-metal's globals build specifically, or maybe AMD as well). That's why I used to change it manually to the lines you've replaced here:

```js
(function (global, factory) {
   (factory((global.IncrementalDOM = global.IncrementalDOM || {})));
 }(globals.window, function (exports) { 'use strict';
```